tests/unit/test_synapses.cpp

```diff
@@ -93,6 +99,7 @@ TEST(synapses, expsyn_basic_state)
     ptr->net_receive(3, 1.04);
     EXPECT_EQ(ptr->g[1], 3.14);
     EXPECT_EQ(ptr->g[3], 1.04);
+
```

Comment:
I should cleanup a bit this file and revert it to its original version, since now mechanisms tests are moved to `test_mechanisms.cpp`.